modules/ROOT/pages/cors-policy.adoc

```diff
@@ -4,24 +4,49 @@ include::_attributes.adoc[]
+CORS is a mechanism by which a Web application can access resources that are defined in another domain.  Using CORS, applications invoking JavaScript XMLHttpRequest (XHR) calls from a Web page can interact with resources from non-origin domains.
+
+Typically, Web browsers implement this standard for allowing cross origin requests. Your API is categorized as a public resource by default when you apply this policy. If you want to share your API with only specific resources, you can configure the policy as a restricted resource.
+
+The CORS policy complies with CORS W3C recommendation from 16 January 2014.
+
+*can we add a link to it?*
+
+*<Eva: I saw that you can specify a default group in the UI. What is it? what does it do and what other groups can one create?>*
+
+You must specify the default group before specifying another one. The Default group is not a fallback in the normal sense of a default. In this case, it is only the first group you configure for CORS. -- Eva, not sure about this sentence. What does the fallbacl mean here. Please clarify.
+
+You cannot order the CORS policy. This isn't a technical limitation, but rather by design. No OPTIONS request can reach the backend if CORS is configured, as this request is treated as a Preflight according to the CORS-compliant RFC.
+
+
+
+*Eva again the previous sentence does not make sense*
 
 image::cors-policy.png[]
 
-NOTE: You must specify the default group before specifying another one. The Default group is not a fallback in the normal sense of a default. In this case, it is only the first group you configure for CORS.
+*Eva--do we have any prerequisites or scope and considerations for this policy?*
 
-*Important:* The CORS policy cannot be ordered. This isn't a technical limitation, but rather by design. No OPTIONS request can reach the backend if CORS is configured, as this request is treated as a Preflight according to the CORS-compliant RFC.
+== How This Policy Works
 
-== CORS as an Interceptor
+*Eva we don't have much here.*
 
-In Mule Runtime 4.0, the CORS Interceptor is a element in the HTTP Listener configuration. To be consistent, its configuration is the same as in the policy. Use this feature when the Mule Runtime has not been started with API Gateway capabilities and CORS functionality is needed.
+== Configuring CORS for non-API Gateway Mule Environments
 
-NOTE: Beware that in this scenario, altering the configuration will force an application redeploy.
+If your Mule runtime engine (Mule) is not started with API gateway capabilities and you need to implement the CORS functionality, you can use the CORS Interceptor, which is an element in the HTTP Listener configuration and is available in Mule 4.0.
```

Comment:
Not sure: Should we replace "Mule runtime engine (Mule)" by "runtime"?

Reply:
No again product naming is an editor thing. it is fine as is. Done! Thanks!